spec/features/login_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'rails_helper'
+
+RSpec.describe 'Testing the login page', type: :feature do
+  describe 'Test if the contents in the login page are present' do
+    it 'Test if there is an email input field' do
+      visit new_user_session_path
+      expect(page).to have_selector('input[type=email]')
```

Comment:
- [ ] You made great effort on this project. However your tests must past, and should work in any machine (in your specified versions) in order to get approved.

`rspec spec` failing tests starting from `spec/modules`
`bundle exac rspec` still failing in even `spec/features`

_Please write more description in PR description and [OPTIONAL] screenshot of your tests passing._


<img width="1077" alt="Screen Shot 2022-07-02 at 3 48 53 PM" src="https://user-images.githubusercontent.com/69011963/176997665-bb38fc65-1625-4c70-87c2-fb6caa35c0a1.png">